addons/point_of_sale/static/src/xml/Screens/ProductScreen/ActionpadWidget.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="ActionpadWidget" owl="1">
         <div class="actionpad">
-            <button class="button mobile-more-button" t-if="ui.isSmall" t-on-click="props.onClickMore">
+            <button class="button mobile-more-button" t-if="ui.isSmall and props.showMoreButton" t-on-click="props.onClickMore">
```

Comment:
Instead of introducing new props, you can use the `props.onClickMore` as a condition. If provide, show the `More...` button, otherwise, don't show it.